src/components/ChangePlan/PlanCalculator/PlanAgreement/PlanAgreement.js

```diff
@@ -40,7 +40,7 @@ export const PlanAgreement = ({ planData }) => {
         </p>
       ) : null}
 
-      {planData.plan.type === 'monthly-deliveries' ? (
+      {planData.plan.type !== 'subscribers' ? (
```

Comment:
So the value by email is mapped into the same field? I would prefer being explicit here, like this:
```suggestion
      {planData.plan.type === 'monthly-deliveries' && planData.plan.type === 'prepaid'  ? (
```
We have several plans, `subscribers`, `monthly-deliveries` `agencies`, `prepaid`, when I read planData.plan.type === 'monthly-deliveries' I have to think twice to understand that you are not including agencies or exactly to which plans you are pointing out.